26-register_registration/resource_types/research_data/dataset.feature

```diff
@@ -0,0 +1,92 @@
+Feature: Dataset
+
+	In order to document my findings in journal articles
+	As a User
+	I want to be able to publish my own and others datasets
+
+	In order to document my findings in journal articles
+	As a User
+	I want to be able to link published datasets to my published journal articles
+
+	Background:
+		Given User selects Resource type "Research Data"
+		And they select Dataset as subtype
+
+	@9141
+	Scenario: User sees information about data that is illegal to publish on this service
+		When the User has selected to register a Dataset
+		Then the User sees information about data that is illegal to publish on this service
+
+	@9141
+	Scenario: User confirms to register data legal to publish
+		Given User sees information about data that is illegal to publish on this service
+		When the User confirms that the data intended to be published is legal 
+		Then the dialog is closed 
+	
+	@9141
+	Scenario: User confirms to register data illegal to publish
+		Given User sees information about data that is illegal to publish in this service
+		When the User confirms that the data intended to be published is illegal 
+		Then the User is prohibited to publish the Registration
+		And any registered data is stored as a draft
+		And the User sees the standard user support dialog where the user can ask for assistance
+
+	@9140
+	Scenario: User is informed about further support and registration process
+		Given User confirms to register data illegal to publish
+		When the User has submitted a user support request
+		Then the User is informed that the registration is stored
+		And can be completed at any later stage
+		And answer to the user support request will be visible on "My page"
+
+	@9140
+	Scenario: User adds zero or more geographical data to the dataset
+		Given User confirms to register data legal to publish
+		When the User writes some free text geographical data
+		# Future: Add support for land, region, county and map reference (line, rectangle, point)
+		Then stored it
+
+	@9140
+	Scenario: User adds zero or more use-references to resource published in NVA
+		Given User confirms to register data legal to publish
+		When the user enters some text, a matching list of published results are displayed 
+		And the User may select one result
+		Then store it as depended on this result
+
+	@9140
+	Scenario: User adds zero or more related-references to resource published in NVA
```

Comment:
Her står det `resource` mens lenger ned i samme scenario står det `result`. Antar dette er det samme som vi har kalt `Registration` de fleste andre steder. Burde bruke faste begrep for dette

Reply:
Tja, vi er på ein tynn linje mellom ulike nyanser her. 
Dette senario sei "to resource published in NVA", men det er første steg på veien. Om vi lar overskrifta styre, så kan vi endre til Registration. Men, intensjonen er at vi i neste iterasjon av dette senario også skal tillate å lenke til eksternt publiserte forskingsresultat - da blir Registration feil.

Reply:
Har oppdatert etter diskusjon - trur det er bra no.